binr/r2r/r2r.c

```diff
@@ -97,10 +97,13 @@ static bool r2r_chdir(const char *argv0) {
 }
 
 static void r2r_test_run_unit(void) {
-	system ("make -C ../unit all run");
+	system ("make -C unit all run");
```

Comment:
By the way, I think it should resolve also the `$MAKE` env var.

Reply:
well. we shouldnt use MAKE at all. this function must be reimplemented in pure C, just iterate over all the unit test executables and run them one by one. also compile them if they are not generated

Reply:
we can probably wait until i do this in this PR to merge it

Reply:
No, imo r2r shouldn't touch unit tests at all. But if you want this so bad it's ok to me since I don't have to use it myself.

Reply:
well, r2r was suposed to run the unit tests before this pr and it wasnt working after moving the tests/new/db to tests/db